perma_web/frontend/components/LinkCount.vue

```diff
@@ -14,7 +14,7 @@ const getLinksRemainingText = () => {
 
         case 'metered':
         default:
-            return `You have <span class="links-remaining">${links_remaining}</span> remaining Perma Links${linkRenewalPeriod}</span><br />`
+            return `You have <span class="links-remaining">${globalStore.linksRemaining}</span> remaining Perma Links${linkRenewalPeriod}</span><br />`
```

Comment:
If we are going to refer to a variable in more than one component, it felt like it might make sense to include it in the store.